src/app/finished-analysis-view/finished-analysis-view/finished-analysis-view.component.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * finished-analysis-view.component.ts
+ * Component for visualization of all aspects of the result.
+ *
+   Copyright 2018 Grzegorz Mrukwa
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import {
+  Component,
+  OnInit,
+} from '@angular/core';
+
+import { QuerySchemaDownloadService } from '../query-schema-download.service';
+import { AspectDescription } from '../../aspect-view/aspect-description';
+import {
+  ActivatedRoute,
+  ParamMap,
+} from '@angular/router';
```

Comment:
You introduce this style of import or is it mistake?

Reply:
I introduced this style, since it is much more git-friendly. It causes much less conflicts.